### DISCUSSION TITLE
Edge border Radius

### DISCUSSION THREAD

**Discussion by nolderosw** [bcakmakoglu/vue-flow#290] [Q&A] [answered]
How to set edge border radius via props?
Im trying like this but unsuccessfully:
![image](https://user-images.githubusercontent.com/21163236/193342639-404e6f3c-af00-4cbc-be94-3865661df339.png)

![image](https://user-images.githubusercontent.com/21163236/193342701-88eb1ee4-9c08-4b43-8c23-3f2e5153768a.png)

**Comment by bcakmakoglu** [comment, marked as answer]
Mh... currently you can't pass it to the default edges; I will allow the prop to be used later on.

Instead you can just use either a custom edge component and use `getSmoothstepPath` or use the `SmoothstepEdge` exported by the library and just feed it the props

```
<template #edge-custom-step="props">
	<SmoothstepEdge v-bind="props" />
</template>

**Reply by nolderosw** [reply]
Thank you again, you are a pretty good library owner! Its a good release pass it to default edges hahah!

Best Regards!
